MachineLearning/ibmdbpy_NaiveBayes.ipynb

```diff
@@ -0,0 +1,957 @@
+    " \n",
+    "The ibmdbpy project provides a Python interface for data manipulation and access to in-database algorithms in IBM Db2 Warehouse. It accelerates Python analytics by seamlessly pushing operations written in Python into the underlying database for execution, thereby benefitting from in-database performance-enhancing features, such as columnar storage and parallel processing. For more details about ibmdbpy, please refer to the [documentation](https://pythonhosted.org/ibmdbpy/index.html) and to the dedicated [Git repository](https://github.com/ibmdbanalytics/ibmdbpy/tree/master/ibmdbpy). This notebook provides you with an overview of ibmdbpy functionalities. \n",
+    "\n",
+    " \n",
+    "__Prerequisites__\n",
+    "* Db2 account: see [IBM Cloud](https://cloud.ibm.com/login) or [Db2 Warehouse](https://www.ibm.com/support/knowledgecenter/en/SSCJDQ/com.ibm.swg.im.dashdb.kc.doc/welcome.html)\n",
+    "* Db2 driver: learn more on [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/en/SSFMBX/com.ibm.swg.im.dashdb.doc/connecting/connect_applications_by_type.html) and see [IBM Support](https://www.ibm.com/support/pages/db2-jdbc-driver-versions-and-downloads)\n",
+    "* Having installed the [ibmdbpy](https://pypi.org/project/ibmdbpy/) python library with pip: \n",
+    "> pip install ibmdbpy \n",
+    "* Optional dependency for JDBC is the [jaydebeapi](https://pypi.org/project/JayDeBeApi/) library. Run the following command to install ibmdbpy, as well as the dependencies for the JDBC feature:\n",
+    "> pip install ibmdbpy[jdbc]\n",
+    "\n",
+    "__About naive Bayes classification__\n",
+    "\n",
+    "The Naive Bayes algorithm is a type of probabilistic ML classification algorithm. It is based on the Bayes theorem and on the assumption that predictors are independent. In practice, this is not really the case but naive Bayes classifiers perform quite well on tasks such as sentiment analysis, spam filtering and product recommendation.\n",
+    "More on the [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSCJDQ/com.ibm.swg.im.dashdb.analytics.doc/doc/r_naive_bayes.html).\n",
+    "\n",
+    "__Contents__\n",
+    "\n",
+    "This notebook shows you how to perform naive Bayes classification with ibmdbpy on the iris data sample.\n",
+    "\n",
+    "__1. Get started__\n",
+    "* Imports\n",
+    "* Establish a JDBC connection to Db2\n",
+    "* Load sample data\n",
+    "\n",
+    "__2. Perform Naive Bayes Classification with ibmdbpy__\n",
+    "* Define an indexer\n",
+    "* Build the model\n",
+    "* Analyze the results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Get started"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Imports__\n",
+    "\n",
+    "As usual, we import jaydebeapi in order to establish a JDBC connectio to Db2, and import some modules from ibmdbpy."
```

Comment:
```suggestion
    "As usual, we import jaydebeapi in order to establish a JDBC connection to Db2, and import some modules from ibmdbpy."
```

Reply:
actually this sentence has to be deleted, jaydebeapi is no more imported explicitly in this cell